launch/move_base.launch

```diff
@@ -8,26 +8,24 @@
 
 	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true" unless="$(arg with_mux)">
 		<!-- default:20.0. with this value dwa planner fails to find a valid plan a lot more -->
-                <rosparam file="$(find strands_movebase)/strands_movebase_params/move_base_params.yaml" command="load"/>
+        <rosparam file="$(find strands_movebase)/strands_movebase_params/move_base_params.yaml" command="load"/>
```

Comment:
you should really try not to reformat things that are not effectively changed...